multi-agent/lib/mas/resources/builtin_aware_service.py

```
@@ -0,0 +1,537 @@
+        inner: CoreResourceService,
+        *,
+        descriptor_repo: BuiltinResourceDescriptorRepository,
+        builtin_user_config_repo: Optional[BuiltinUserConfigRepository] = None,
+        admin_lock_reader: Optional[AdminEditLockReader] = None,
+    ) -> None:
+        self._inner = inner
+        self._descriptors = descriptor_repo
+        self._user_configs = builtin_user_config_repo
+        self._admin_lock_reader = admin_lock_reader
+
+    def __getattr__(self, name: str) -> Any:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `__getattr__`

(ANN401)